plugins/chat/assets/javascripts/discourse/components/chat-replying-indicator.js

```diff
@@ -13,37 +13,32 @@ export default class ChatReplyingIndicator extends Component {
   @tracked users = [];
 
   @action
-  async subscribe() {
-    this.presenceChannel = this.presence.getChannel(this.channelName);
-    this.presenceChannel.on("change", this.handlePresenceChange);
-    await this.presenceChannel.subscribe();
-  }
-
-  @action
-  async resubscribe() {
-    await this.teardown();
+  async updateSubscription() {
+    await this.unsubscribe();
     await this.subscribe();
   }
 
   @action
-  handlePresenceChange(presenceChannel) {
-    this.users = presenceChannel.users || [];
+  async subscribe() {
+    this.presenceChannel = this.presence.getChannel(this.channelName);
+    await this.presenceChannel.subscribe();
+    this.users = this.presenceChannel.users;
+    this.presenceChannel.on("change", this.handlePresenceChange);
   }
 
   @action
-  async handleDraft() {
-    if (this.args.channel.isDraft) {
-      await this.teardown();
-    } else {
-      await this.resubscribe();
+  async unsubscribe() {
+    this.users = [];
+
+    if (this.presenceChannel.suscribed) {
```

Comment:
```suggestion
    if (this.presenceChannel.subscribed) {
```

Reply:
Im starting to like more and more types 🙃